1e/README.md

```diff
@@ -0,0 +1,168 @@
+# 1E
+
+## Overview
+
+At [1E][1], we reimagine how technology serves people and create new ways for IT to shape the future of work.
```

Comment:
Could you provide 1-2 sentences on 1E does? We try to avoid marketing copy in the overview.

Reply:
Added sentence on what do we do at 1E